Library/Homebrew/cmd/audit.rb

```diff
@@ -1,87 +1,76 @@
-#:  * `audit` [`--strict`] [`--online`] [<formulae>]:
+#:  * `audit` [`--strict`] [`--online`] [`--display-cop-names`] [<formulae>]:
 #:    Check <formulae> for Homebrew coding style violations. This should be
 #:    run before submitting a new formula.
 #:
 #:    If no <formulae> are provided, all of them are checked.
 #:
-#:    If `--strict` is passed, additional checks are run. This should be used
-#:    when creating for new formulae.
+#:    If `--strict` is passed, additional checks are run, including Rubocop
```

Comment:
Well, I'll be a monkey's uncle. IMDB says the movie is "RoboCop", too. Fixed.

I left capitalization of class names as `RubocopResults` and the like, following the Java CamelCase convention that internal capitalization styling is ignored for single words and acronyms, e.g. `JsonParser`, not `JSONParser`.